src/features/homepage/components/HomepageStack.tsx

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from "react-native"
+import { createStackNavigator } from '@react-navigation/stack'
+import { Screens } from 'constants'
+import { Homepage } from './Homepage'
+import { HeadLineDetails } from './HeadLineDetails'
+import { Notifications } from './Notifications'
+
+const HomepageStack = (): JSX.Element => {
+    const Stack = createStackNavigator()
```

Comment:
createStackNavigator() needs the type of the param list  you created

`createStackNavigator<HomeStackParamList>()`